.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,21 +1,42 @@
-### Purpose of PR:
-Please give a short description of what this PR is for.
+### Purpose of PR  
+Describe what this PR does.
 
+### Linked Issues  
+Add links to related issues.
+- Fixes MAHOUT-XXXX  
```

Comment:
This is a nit, but since you need to uncomment the lines in the other file anyway, replace this line with

`Closes #123`